src/app/account-app/account-receipt.component.ts

```diff
@@ -0,0 +1,53 @@
+// --------- BEGIN RUNBOX LICENSE ---------
+// Copyright (C) 2016-2019 Runbox Solutions AS (runbox.com).
+//
+// This file is part of Runbox 7.
+//
+// Runbox 7 is free software: You can redistribute it and/or modify it
+// under the terms of the GNU General Public License as published by the
+// Free Software Foundation, either version 3 of the License, or (at your
+// option) any later version.
+//
+// Runbox 7 is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Runbox 7. If not, see <https://www.gnu.org/licenses/>.
+// ---------- END RUNBOX LICENSE ----------
+
+import { Component } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { RunboxMe, RunboxWebmailAPI } from '../rmmapi/rbwebmail';
+
+@Component({
+    selector: 'app-account-receipt-component',
+    templateUrl: './account-receipt.component.html',
+})
+export class AccountReceiptComponent {
+    receipt: any = { products: [] };
+    me: RunboxMe = new RunboxMe();
```

Comment:
in this case you will overwrite me as soon as the subcribe is done. I'm wondering if it would be better to use an asyncsubject here instead of starting with an empty RunboxMe object. In fact `me` is already an async subject that you can use directly from here: https://github.com/runbox/runbox7/blob/master/src/app/rmmapi/rbwebmail.ts#L166

Reply:
I think I put it here as an empty object so that the HTML rendering doesn't break on an undefined variable when I do `{{ me.username }}` and alike in the template.

“Make sure the template isn't broken before the data arrives” is a common problem I have – how would I solve it elegantly here?

Reply:
Done in 17c87e3.